input/fsh/mhd.fsh

```diff
@@ -269,4 +269,234 @@ Description: "CH MHD SubmissionSet Comprehensive"
 * entry.item only Reference($ch-mhd-documentreference-comprehensive)
 * entry.item MS
 * entry.item ^type.aggregation[0] = #referenced
-* entry.item ^type.aggregation[+] = #bundled
\ No newline at end of file
+* entry.item ^type.aggregation[+] = #bundled
+
+Profile: CHMhd1UpdateDocumentMetadataTransactionRequest
+Id: ch-mhd-1-updatedocumentmetadatatransactionrequest
+Parent: Bundle
+Title: "Process"
+Description: "A profile for Update Document Metadata (CH:MHD-1) transaction request"
+* type = #transaction
+* entry.request 1..1 MS
+* entry.request.method = #PUT 
+* entry.request.url 1..1 
+* entry ^slicing.discriminator[0].type = #profile
+* entry ^slicing.discriminator[0].path = "resource"
+* entry ^slicing.rules = #open
```

Comment:
That profile doesn't prevent submitting other resources in the transaction (which could trigger other ITI transactions) 😥